src/main/java/budgetbuddy/model/attributes/Name.java

```diff
@@ -19,7 +19,7 @@
      */
     public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
 
-    public final String fullName;
+    public final String name;
```

Comment:
Just a little curious why this file is public

Reply:
It's a remnant of addressbook's design. Their Name, Address, Email, etc. all have public variables. I figured it was fine since they're all final, so it wouldn't matter if other classes could access them directly. But I could change it to private?

Reply:
Hmm, not sure how we should go about this. I know that some other classes do reference this field directly though.